Index: 0008_插图.78749
===================================================================
--- 0008_插图.78749	[object Object]
+++ 0008_插图.78749
@@ -1,16 +1,16 @@
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94156.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94157.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94158.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94159.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94160.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94161.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94162.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94163.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94164.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94165.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94166.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94167.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94168.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94169.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94170.jpg"/>
-<img src="http://pic.wenku8.com/pictures/2/2172/78749/94171.jpg"/>
\ No newline at end of file
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94156.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94157.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94158.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94159.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94160.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94161.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94162.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94163.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94164.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94165.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94166.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94167.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94168.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94169.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94170.Jpg」/】
+【Img Src=「Http：//Pic.wenku8.Com/Pictures/2／2172／78749／94171.Jpg」/】
